testing/maven/src/main/java/org/commonjava/maven/galley/testing/maven/GalleyMavenFixture.java

```diff
@@ -592,7 +593,7 @@ public GalleyMavenFixture withPomReader( final MavenPomReader pomReader )
         return this;
     }
 
-    public GalleyMavenFixture withPluginDefaults( final StandardMaven304PluginDefaults pluginDefaults )
+    public GalleyMavenFixture withPluginDefaults( final StandardMaven350PluginDefaults pluginDefaults )
```

Comment:
There is already a `withPluginDefaults ( final MavenPluginDefaults pluginDefaults )` so perhaps this could be removed. Note there is also a matching function in CartographerCoreBuilder I believe.

Reply:
:+1:

Reply:
Agreed, MavenPluginDefaults is the right parameter type.